web/thesauruses/c/17/functions.json

```diff
@@ -0,0 +1,72 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "functions"
+    },
+    "concepts": {
+        "void_functions_no_parameters": {
+            "code": "void foo() { return; }",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and takes no parameters"
+        },
+        "void_function_with_parameters": {
+            "code": "void foo(char c, int i, double d) { return; }",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and that takes 1 or more defined parameters"
+        },
+        "void_function_with_keyword_parameters": {
+            "code": "unknown",
```

Comment:
You shouldn't leave this unknown. Try to research it, ask, or just say `"not-implemented": true` if you're unsure and maybe someone can correct it.